### DISCUSSION TITLE
Appsmith - OneAdmin process

### DISCUSSION THREAD

**Discussion by benjamincoughlin** [twilio-labs/paste#3912] [General] [answered]
The front end development for OneAdmin is moving to Appsmith.

This is going to affect all OneAdmin projects & updates going forward and a project within O&C is the first one to utilize the 3rd party software.

How should we be addressing Appsmith (and any other 3rd party development tools/frameworks) that do not support Paste directly.

- Builder.io for Console is starting soon
- Persona reskinning as it does not support Design System integration
- Courier for notifications
- etc.

Appsmith - [homepage](https://www.appsmith.com/)
Appsmith - [Github Repo](https://github.com/appsmithorg/appsmith)
Appsmith - [Storybook](https://design-system.appsmith.com/?path=/docs/docs-introduction--documentation)
Appsmith - [Figma File Design System](https://www.figma.com/design/aHJfaj4JuTGluy3dMy0gmE/Appsmith-Design-System-2.0-(Community)?node-id=3304%3A15139&t=aRW9pKpGdXbEqYYa-1)

**Comment by benjamincoughlin** [comment]
Anusha built out an really great comparison of Paste to Persona. I am trying to build this out for Paste -> Appsmith.
Here is the [Figma File](https://www.figma.com/design/2vbGZkmkDEfDmDqwrg2Bnc/Persona-%3C%3E-Paste-design-system-audit?node-id=1%3A369363&t=1m3npY6jNko7A4Qh-1)

**Comment by serifluous** [comment, marked as answer]
Going to keep this thread open for any follow-up discussions that might come up about 3rd party tools. But marking it as answered for now.

Folks can reference the #3rd-party-tools channel for internal info.
